src/common/schemas/input/compute-ledger-hash.json

```diff
@@ -5,7 +5,8 @@
   "properties": {
     "ledger": {
       "$ref": "getLedger",
-      "description": "The ledger header to hash."
+      "description": "The ledger header to hash.",
+      "position": 0
```

Comment:
This is not a very robust solution because it can get out of sync with the actual method signatures. I think it would be better to use the same technique that jayson uses to get the method parameter ordering.